lms/djangoapps/badges/migrations/0002_data__migrate_assertions.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import json
+
+import os
+from django.db import migrations, models
+
+
+def forwards(apps, schema_editor):
+    """
+    Migrate the initial badge classes, assertions, and course image configurations from certificates.
```

Comment:
In particular, i looked for network access here and didn't see it, but we accessing files on disk, yes?